shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -2642,6 +2642,9 @@ private void updateRecursively(
         if (globalTransform == null) {
           globalTransform = new float[16];
         }
+        if (transform == null) {
+          transform = new float[16];
```

Comment:
Shouldn't this and the thing above be initialized to an identity transformation?

Reply:
cc @Hixie who may have context as the author of the original null check that initialized all matrix elements to zer.

Reply:
better than crash

Reply:
> cc @Hixie who may have context as the author of the original null check that initialized all matrix elements to zero.

Hope merge, plz

Reply:
This should be initialized during the SemanticsNode.updateWith. The only possible case I can think of is the framework sent a semantics node with id child that is not in the update list. This feels like a bug in the framework side. Do you have a reproducible code?